tests/integration_tests/test_postprocessing.py

```diff
@@ -59,7 +59,7 @@ def random_set_logits(*args, num_predict_samples, vocab_size, pct_positive, **kw
     ],
 )
 @pytest.mark.parametrize("distinct_values", [(False, True), ("No", "Yes")])
-def test_binary_predictions(tmpdir, backend, distinct_values, ray_cluster_2cpu):
+def test_binary_predictions(tmpdir, backend, distinct_values, ray_cluster_4cpu):
```

Comment:
I would prefer to keep it to 2 cpu, as it should work. If it's not, we should figure out what's causing the spike in resource demands.

Reply:
Agreed, this was just a temporary change to make sure all the tests work with the new trainer